README.md

```diff
@@ -29,6 +29,7 @@ Have any questions? Feel free to ask in the dedicated [`bevy_enhanced_input` cha
 
 | bevy        | bevy_enhanced_input |
 | ----------- | ------------------- |
+| 0.17.0-rc      | 0.19.0-rc           |
```

Comment:
not bothering with formatting since the table would need to be formatted back when the RC is over